modules/light-clients/08-wasm/keeper/genesis.go

```diff
@@ -29,7 +31,7 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) types.GenesisState {
 	// Grab code from wasmVM and add to genesis state.
 	var genesisState types.GenesisState
 	for _, codeHash := range codeHashes {
-		code, err := k.wasmVM.GetCode(codeHash)
+		code, err := k.wasmVM.GetCode(wasmvmtypes.Checksum(codeHash))
```

Comment:
Adding the type alias means we need to do some type conversions when calling into the vm.

The `Checksum` type in `wasmvm` is also just an alias on `[]byte`